src/auth/auth-router.js

```diff
@@ -0,0 +1,47 @@
+const express = require('express');
+const authManager = require('./auth-manager');
+
+const authRouter = express.Router();
+
+// Get the current User's role.
+authRouter.get('/', authManager.verify, (req, res) => {
```

Comment:
errors from `authManager.verify` are not caught it seems. When this throws an error, a 500 status code and a html page are served up instead of a JSON body with the error.